tests/phpunit/class-wc-stripe-connect-test.php

```
@@ -119,4 +119,47 @@ public function provide_save_stripe_keys_defaults_ocs_for_new_installs(): array
+
+		$this->assertStringContainsString( 'wc_stripe_connect_error=expired_nonce', $redirected_to );
+		$this->assertStringNotContainsString( 'wcs_stripe_code', $redirected_to );
+		$this->assertStringNotContainsString( 'wcs_stripe_state', $redirected_to );
+
+		unset(
+			$_GET['wcs_stripe_code'],
+			$_GET['wcs_stripe_state'],
+			$_GET['wcs_stripe_mode'],
+			$_GET['_wpnonce']
+		);
+	}
```

PHPMD (2.15.0)

[error] test_maybe_handle_redirect_redirects_with_error_on_invalid_nonce accesses the super-global variable $_GET. (undefined)

(Superglobals)